packages/account/src/Components/error-component/error-component.tsx

```diff
@@ -4,10 +4,10 @@ import { localize } from '@deriv/translations';
 import { routes } from '@deriv/shared';
 
 type TErrorComponent = {
-    header: React.ReactElement;
-    message: React.ReactNode;
-    redirect_label: React.ReactElement;
-    redirectOnClick: () => void;
+    header: JSX.Element | string;
```

Comment:
why did you change it here to JSX.Element? is React.Element not working here?

Reply:
I get this error ` Type 'string | Element' is not assignable to type 'ReactElement`